lib/ReactGridLayout.jsx

```diff
@@ -152,11 +166,33 @@ var ReactGridLayout = React.createClass({
     this.setState({width: width});
   },
 
-  onDragStart(i, e, {element, position}) {
-    // nothing
+  /**
+   * When dragging starts
+   * @param {Number} i Index of the child
+   * @param {Number} x X position of the move
+   * @param {Number} y Y position of the move
+   * @param {Element} element The current dragging DOM element
+   * @param {Object} position Drag information
+   */
+  onDragStart(i, x, y, {e, element, position}) {
+    var layout = this.state.layout;
+    var l = utils.getLayoutItem(layout, i);
+    if (this.props.onDragStart(layout, this.state.layout, l, null, {e, element, position}) === false) {
```

Comment:
Of course if we did that we'd have to be consistent among all the other callbacks as well. With this code as-is the placeholder would quickly be replaced by the drag handler below.